Roles/AddOns/Common/Sloth.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AmongUs.GameOptions;
+using static TOHE.Options;
+
+namespace TOHE.Roles.AddOns.Common;
+
+public class Sloth : IAddon
+{
+    private const int Id = 29700;
+    public AddonTypes Type => AddonTypes.Harmful;
+
+    private static OptionItem OptionSpeed;
+
+    public void SetupCustomOption()
+    {
+        SetupAdtRoleOptions(Id, CustomRoles.Sloth, canSetNum: true, tab: TabGroup.Addons, teamSpawnOptions: true);
+        OptionSpeed = FloatOptionItem.Create(Id + 10, "SlothSpeed", new(25f, 75f, 5f), 50f, TabGroup.Addons, false)
```

Comment:
It seems you wanted to make its speed lower, but it seems here it is faster than even Flash itself

In `FloatOptionItem` and in `IntegerOptionItem` set's values like that:

new(0.25f, 1.5f, 0.25f), 0.75f
new(min, max, changedValue), defaultValue

Reply:
Woops! That obviously wasn't the intention haha, should be fixed now